docs/support/tutorials/python-usage-guide.md

```diff
@@ -0,0 +1,254 @@
+		including how to convert Docker containers to Apptainer containers.
+
+=== "conda"
+
+	Conda is easy to use and flexible, but it usually creates a huge number of files which
+	is incompatible with shared file systems. The excess of files can cause
+	very slow library imports and,
+	in the worst case, slowdowns in the whole file system. Therefore,
+	**CSC has deprecated the use of conda for direct installations on
+	supercomputers**. However, you can still create and use
+	[containerized](../../computing/containers/overview.md)
+	conda environments.
+
+	1. The most straightforward way to create a containerized conda
+	   environment is by using the [Tykky container
+	   wrapper](../../computing/containers/tykky.md).
+	   To find out how to easily containerize your environment,
+	   see the [Tykky instructions for conda-based
+	   installations](../../computing/containers/tykky.md#conda-based-installations).
+
+	2. An alternative to using Tykky is creating a conda environment
+	   inside a custom Apptainer container. This is a practical choice if, for
+	   example, you know of a suitable ready-made Apptainer or Docker container.
+	   For more information about using Apptainer containers, please see the
+	   related documentation:
+
+		* [Running Apptainer containers](../../computing/containers/run-existing.md)
+		* [Creating Apptainer containers](../../computing/containers/creating.md),
+	  	including how to convert Docker containers to Apptainer containers.
+
+	The [CSC conda tutorial](./conda.md) describes in more detail
+	what conda is and how to use it. Some parts of the tutorial may
+	also be helpful for Tykky installations.
+
+## Python development environments
+
+Python code can be edited with a console-based [text editor directly on the
+supercomputer](./env-guide/text-and-image-processing.md).
+Codes can also be edited on your local machine and copied to the supercomputer
+with [scp](../../data/moving/scp.md) or [graphical file transfer
+tools](../../data/moving/graphical_transfer.md).
+You can also [edit Python scripts in Puhti from your local
+PC](./remote-dev.md) with some code editors like Visual
+Studio Code.
+
+Finally, several graphical programming environments can be used directly on the
+supercomputer, such as Jupyter Notebooks, Spyder and Visual Studio Code, through
+the [Puhti web interface](../../computing/webinterface/index.md).
+
+### Jupyter Notebooks
+
+[Jupyter Notebooks](https://jupyter.org/) allows one to run Python code via a web
+browser running on a local PC. The notebooks can combine code, equations,
+visualizations and narrative text in a single document. Many of our modules, including
+[python-data](../../apps/python-data.md), the deep learning modules and
+[geoconda](../../apps/geoconda.md)
+include the Jupyter notebook package. See the tutorial [how to set up and connect to
+a Jupyter Notebook](./rstudio-or-jupyter-notebooks.md) for using
+Jupyter in CSC environment.
```

Comment:
If you don't read carefully, on the linked page `rstudio-or-jupyter-notebooks.md` you can easily miss the fact that the easiest (and recommended) way of launching Jupyter Notebook is via Open Ondemand, not via ssh-tunneling etc. We should probably here link directly to the OOD apps.

Reply:
The section now links to OOD and ssh-tunneling separately, and OOD is presented as the recommended option.